.github/ISSUE_TEMPLATE/hotfix-release.md

```diff
@@ -0,0 +1,44 @@
+---
+name: k6 Hotfix release
+about: Bugs or security issues have been identified and they requires a patch release.
+title: 'k6 hotfix release v<major>.<minor>.<patch>'
+labels: ["release"]
+---
+
+**Release Date**:
+
+25th May 2024 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
+
+## Release Activities
+
+Note, the assumption is a minor branch release already exists from generated from the latest minor release. It has the name in the form `release-v<major>.<minor>.x`.
+
+#### Documentation
+
+- [ ] If required, open and merge pull-request from `main` applying the changes to the affected version.
+
+#### In k6 repository, if the defect affects the latest release and the `master` branch
+
+- [ ] Open, get approvals and merge on `master` the patch for the defect.
+- [ ] Open, get approvals and merge on `master` the release notes. Ensure that all merged pull-requests are referenced.
+- [ ] Switch on the release branch and cherry-pick the following changes:
+      1. the patch for the defect
+      2. the release notes
```

Comment:
TODO: Explore the backport GitHub Action from https://github.com/grafana/grafana-github-actions, any idea on where to add this as an item to work on in the future? 🤔

Reply:
I guess the best way is to open an issue for it. Can you take care of it, please?